src/polymer/analyze-properties.ts

```diff
@@ -34,14 +34,19 @@ import {ScannedPolymerProperty} from './polymer-element';
  * @param document The containing JS document.
  */
 export function analyzeProperties(
-    node: estree.Node, document: JavaScriptDocument): ScannedPolymerProperty[] {
+    node: babel.Node, document: JavaScriptDocument): ScannedPolymerProperty[] {
   const analyzedProps: ScannedPolymerProperty[] = [];
 
-  if (node.type !== 'ObjectExpression') {
+  if (!babel.isObjectExpression(node)) {
     return analyzedProps;
   }
 
   for (const property of node.properties) {
+    // TODO(usergenic): SpreadProperty doesn't have value so I have to skip
```

Comment:
issue. since this pops up a lot, it'll be nice to be able to search for the issue number in source.

Reply:
I added issue.  All we have to do is search for SpreadProperty.